reporter-influxdb/src/test/java/io/ultrabrew/metrics/reporters/influxdb/InfluxDBClientTest.java

```diff
@@ -147,6 +147,22 @@ public void testWriteSplitting() throws IOException {
     }};
   }
 
+  @Test
+  public void testNullStrings() throws Exception {
+    new Expectations() {{
+      
+    }};
+    client.write(null, new String[] { "host", "web01" }, new String[] { "temp", "80" }, 1534055562000000003L);
+    client.write("cpu_load_short", new String[] { null, "web01" }, new String[] { "temp", "80" }, 1534055562000000003L);
+    client.write("cpu_load_short", new String[] { "host", null }, new String[] { "temp", "80" }, 1534055562000000003L);
+    client.write("cpu_load_short", new String[] { "host", "web01" }, new String[] { null, "80" }, 1534055562000000003L);
+    client.write("cpu_load_short", new String[] { "host", "web01" }, new String[] { "temp", null }, 1534055562000000003L);
```

Comment:
You should probably test a case where not all the data is invalid to verify the buffer position rollback works correctly.

Reply:
good point.